Resources/Prototypes/_CorvaxNext/Recipes/Cooking/meal_recipes.yml

```
@@ -35,7 +35,7 @@
 - type: microwaveMealRecipe
   id: RecipeOkroshkasoup
```

```suggestion
  id: RecipeOkroshkaSoup
```

Может так сделать всем рецептам?